app/views/layouts/application.html.slim

```diff
@@ -7,6 +7,6 @@ html
     = metamagic
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
-  body
+  body.text
```

Comment:
Mmm.. I wouldn't style the body directly. Use the text class when you are styling the actual text you need to style. In this case, you should add this class to the text you need it to have it

Reply:
But since you already did that, I think this class is not actually needed :smile: